diff --git a/.gitignore b/.gitignore
index e4fdbdb..917a7df 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1 +1,6 @@
-*.import
+# Keep *.import files - They are used to define import settings
+# The .import folder on the other hand has files that will change (regenerate)
+# on every machine that opens the project, and are just related to that machine
+# rathen than project settings.
+.import/
+
diff --git a/assets/segment.png.import b/assets/segment.png.import
new file mode 100644
index 0000000..b7a3594
--- /dev/null
+++ b/assets/segment.png.import
@@ -0,0 +1,34 @@
+[remap]
+
+importer="texture"
+type="StreamTexture"
+path="res://.import/segment.png-44a0da1eb45f4b8426efcea1236d70d2.stex"
+metadata={
+"vram_texture": false
+}
+
+[deps]
+
+source_file="res://assets/segment.png"
+dest_files=[ "res://.import/segment.png-44a0da1eb45f4b8426efcea1236d70d2.stex" ]
+
+[params]
+
+compress/mode=0
+compress/lossy_quality=0.7
+compress/hdr_mode=0
+compress/bptc_ldr=0
+compress/normal_map=0
+flags/repeat=0
+flags/filter=true
+flags/mipmaps=false
+flags/anisotropic=false
+flags/srgb=2
+process/fix_alpha_border=true
+process/premult_alpha=false
+process/HDR_as_SRGB=false
+process/invert_color=false
+stream=false
+size_limit=0
+detect_3d=true
+svg/scale=1.0
diff --git a/guy.png.import b/guy.png.import
new file mode 100644
index 0000000..290916d
--- /dev/null
+++ b/guy.png.import
@@ -0,0 +1,34 @@
+[remap]
+
+importer="texture"
+type="StreamTexture"
+path="res://.import/guy.png-64a8d6e0294e356486ef1e77ad36474d.stex"
+metadata={
+"vram_texture": false
+}
+
+[deps]
+
+source_file="res://guy.png"
+dest_files=[ "res://.import/guy.png-64a8d6e0294e356486ef1e77ad36474d.stex" ]
+
+[params]
+
+compress/mode=0
+compress/lossy_quality=0.7
+compress/hdr_mode=0
+compress/bptc_ldr=0
+compress/normal_map=0
+flags/repeat=0
+flags/filter=false
+flags/mipmaps=false
+flags/anisotropic=false
+flags/srgb=2
+process/fix_alpha_border=true
+process/premult_alpha=false
+process/HDR_as_SRGB=false
+process/invert_color=false
+stream=false
+size_limit=0
+detect_3d=true
+svg/scale=1.0
diff --git a/icon.png.import b/icon.png.import
new file mode 100644
index 0000000..96cbf46
--- /dev/null
+++ b/icon.png.import
@@ -0,0 +1,34 @@
+[remap]
+
+importer="texture"
+type="StreamTexture"
+path="res://.import/icon.png-487276ed1e3a0c39cad0279d744ee560.stex"
+metadata={
+"vram_texture": false
+}
+
+[deps]
+
+source_file="res://icon.png"
+dest_files=[ "res://.import/icon.png-487276ed1e3a0c39cad0279d744ee560.stex" ]
+
+[params]
+
+compress/mode=0
+compress/lossy_quality=0.7
+compress/hdr_mode=0
+compress/bptc_ldr=0
+compress/normal_map=0
+flags/repeat=0
+flags/filter=true
+flags/mipmaps=false
+flags/anisotropic=false
+flags/srgb=2
+process/fix_alpha_border=true
+process/premult_alpha=false
+process/HDR_as_SRGB=false
+process/invert_color=false
+stream=false
+size_limit=0
+detect_3d=true
+svg/scale=1.0
diff --git a/project.godot b/project.godot
index cb44ff9..b76809f 100644
--- a/project.godot
+++ b/project.godot
@@ -19,6 +19,12 @@ config/name="Plug It In"
 run/main_scene="res://scenes/levels/area.tscn"
 config/icon="res://icon.png"
 
+[display]
+
+window/size/height=576
+window/stretch/mode="2d"
+window/stretch/aspect="keep"
+
 [input]
 
 move_left={
diff --git a/scenes/cord/cord.gd b/scenes/cord/cord.gd
index e8595c4..463514f 100644
--- a/scenes/cord/cord.gd
+++ b/scenes/cord/cord.gd
@@ -1,49 +1,35 @@
 extends Node2D
 
-var SOFTNESS = 0.1
-var BIAS = 0.9
-export var length : int
+export var LENGTH : int
+export var SOFTNESS = 0.1
+export var BIAS = 0.9
 
-#export(String, FILE, "*.tscn") var segment_scene
 var segment = preload("res://scenes/cord/cord_segment.tscn")
-var joint = preload("res://scenes/cord/cord_joint.tscn")
-
-var joints = []
 
 func _ready():
-	#build_cord()
-	pass
-	
+	build_cord()
+
+func build_cord():
+	var parent = $anchor
+	for i in range(LENGTH):
+		var child = add_segment(parent)
+		add_joint(parent, child)
+		parent = child
+		# hide the last segment - looks better. Comment out to see difference
+		if i == LENGTH-1:
+			child.get_node('Sprite').hide()
+
 func add_segment(parent):
 	var new_segment = segment.instance()
-	new_segment.position = parent.get_end()
+	new_segment.position = parent.position + parent.get_end()
 	add_child(new_segment)
 	return new_segment
 
 func add_joint(parent, child):
-	var new_joint = joint.instance()
+	var new_joint = PinJoint2D.new()
 	new_joint.position = parent.get_end()
 	new_joint.node_a = parent.get_path()
 	new_joint.node_b = child.get_path()
+	new_joint.softness = SOFTNESS
+	new_joint.bias = BIAS
 	parent.add_child(new_joint)
-	joints.append(new_joint)
-
-func build_cord():
-	joints = []
-	var parent = $anchor
-	for i in range(length):
-		var child = add_segment(parent)
-		add_joint(parent, child)
-		parent = child
-
-func resolve_cord(delta):
-	for joint in joints:
-#		joint.resolve(delta)
-		pass
-
-func _physics_process(delta):
-	resolve_cord(delta)
-
-
-func _on_Timer_timeout():
-	build_cord()
\ No newline at end of file
diff --git a/scenes/cord/cord.tscn b/scenes/cord/cord.tscn
index d2c4a75..3ec5eee 100644
--- a/scenes/cord/cord.tscn
+++ b/scenes/cord/cord.tscn
@@ -7,8 +7,3 @@
 script = ExtResource( 1 )
 
 [node name="anchor" parent="." instance=ExtResource( 2 )]
-
-[node name="Timer" type="Timer" parent="."]
-one_shot = true
-autostart = true
-[connection signal="timeout" from="Timer" to="." method="_on_Timer_timeout"]
diff --git a/scenes/cord/cord_joint.gd b/scenes/cord/cord_joint.gd
deleted file mode 100644
index 6fa80ef..0000000
--- a/scenes/cord/cord_joint.gd
+++ /dev/null
@@ -1,21 +0,0 @@
-extends Node2D
-
-export var node_a : NodePath
-export var node_b : NodePath
-
-var nodeA : Node2D
-var nodeB : Node2D
-
-var MAX_DISTANCE = 1.0
-
-func _ready():
-	nodeA = get_node(node_a)
-	nodeB = get_node(node_b)
-
-func resolve(delta):
-	var distance = nodeB.global_position.distance_to(global_position)
-	if distance > MAX_DISTANCE:
-		var a = global_position
-		var b = nodeB.global_position
-		nodeA.rotation = nodeA.global_position.angle_to_point(b) + (PI/2)
-		nodeB.global_position = global_position
\ No newline at end of file
diff --git a/scenes/cord/cord_joint.tscn b/scenes/cord/cord_joint.tscn
deleted file mode 100644
index d2cbd32..0000000
--- a/scenes/cord/cord_joint.tscn
+++ /dev/null
@@ -1,3 +0,0 @@
-[gd_scene format=2]
-
-[node name="cord_joint" type="PinJoint2D"]
diff --git a/scenes/levels/area.tscn b/scenes/levels/area.tscn
index 8f03afa..5d119c6 100644
--- a/scenes/levels/area.tscn
+++ b/scenes/levels/area.tscn
@@ -1,14 +1,20 @@
 [gd_scene load_steps=4 format=2]
 
-[ext_resource path="res://scenes/player/player.tscn" type="PackedScene" id=1]
-[ext_resource path="res://scenes/cord/cord.tscn" type="PackedScene" id=2]
+[ext_resource path="res://scenes/cord/cord.tscn" type="PackedScene" id=1]
+[ext_resource path="res://scenes/player/player.tscn" type="PackedScene" id=2]
 
 [sub_resource type="RectangleShape2D" id=1]
 extents = Vector2( 155, 10 )
 
 [node name="area" type="Node2D"]
 
-[node name="player" parent="." instance=ExtResource( 1 )]
+[node name="cord" parent="." instance=ExtResource( 1 )]
+position = Vector2( 294.968, 282.3 )
+rotation = -1.90241
+LENGTH = 50
+BIAS = 0.5
+
+[node name="player" parent="." instance=ExtResource( 2 )]
 position = Vector2( 468.195, 298.575 )
 
 [node name="platform" type="StaticBody2D" parent="."]
@@ -22,11 +28,6 @@ shape = SubResource( 1 )
 [node name="Polygon2D" type="Polygon2D" parent="platform"]
 polygon = PoolVector2Array( 313.41, 157.088, 624.002, 158.11, 624.683, 177.862, 314.431, 177.522 )
 
-[node name="cord" parent="." instance=ExtResource( 2 )]
-position = Vector2( 254.558, 291.513 )
-rotation = -1.90241
-length = 19
-
 [node name="Camera2D" type="Camera2D" parent="."]
 position = Vector2( 431.689, 344.381 )
 current = true
@@ -37,7 +38,7 @@ margin_left = 185.389
 margin_top = 252.983
 margin_right = 310.389
 margin_bottom = 266.983
-text = "Rope is not working :("
+text = "Rope is kinda working :)"
 
 [node name="controls" type="Label" parent="."]
 margin_left = 557.006
